### ISSUE TITLE
Feature request: Browser sharing uploading files for receiver to share

### ISSUE THREAD

**Issue by trymeouteh** [localsend/localsend#424]
### Is your feature request related to a problem? Please describe.

Allow those without LocalSend to send files to those with LocalSend installed.

### Describe the solution you'd like

LocalSend users starts a LocalSend web server, other users connects to server within their browser and can upload files to send to the LocalSend user.

**Comment by bitjerry** [comment]
Same demand. Since localsend is a cross platform solution, browsers should also be supported.
I hope to have a web version that can start a web server locally when one party starts the localsend client, so that users who do not have localsned installed on the local area network can also transfer files. This is very valuable in the absence of a network.
Technically, it should not be difficult to implement.

**Comment by Tienisto** [comment]
This should be already possible in one way by using the "share via link" option

**Comment by wisinfun** [comment]
> This should be already possible in one way by using the "share via link" option

@Tienisto Hi~ I hope to add settings and enable webserver by default

**Comment by NotOfficer** [comment]
Same demand, would be a great addition.

**Comment by cutejsf** [comment]
> This should be already possible in one way by using the "share via link" option

Can only receive files through a browser, but cannot send files through a browser

**Comment by cutejsf** [comment]
> This should be already possible in one way by using the "share via link" option

@Tienisto 

Can only receive files through a browser, but cannot send files through a browser

**Comment by github2099** [comment]
> Any update on this feature request? It is a very useful feature for sending and receiving files between not LocalSend installed users.

yes absolutely. I have local send installed on my phone but not on my pc. i don't share with my pc that often. it will be handy if i can use browser for the occasional use

**Comment by sahilatahar** [comment]
At least there should be an option send the current link to the phone directly from brower.
